support/doc/production.md

```diff
@@ -104,11 +102,29 @@ server {
   listen [::]:443 ssl http2;
   server_name peertube.example.com;
 
-  # For example with Let's Encrypt
+  # For example with Let's Encrypt (you need a certificate to run https)
   ssl_certificate      /etc/letsencrypt/live/peertube.example.com/fullchain.pem;
   ssl_certificate_key  /etc/letsencrypt/live/peertube.example.com/privkey.pem;
-  ssl_trusted_certificate /etc/letsencrypt/live/peertube.example.com/chain.pem;
-
+  ssl_dhparam /etc/nginx/dhparam.pem; # openssl dhparam -out /etc/nginx/dhparam.pem 4096
+  
+  # Security hardening (as of 11/02/2018)
+  ssl_protocols TLSv1.3;# Requires nginx >= 1.13.0 else use TLSv1.2
```

Comment:
I don't think it's a good idea: https://caniuse.com/#feat=tls1-3

Reply:
`TLSv1.3, TLSv1.2` would be fine.